scenario_test_tools/scripts/example_move_base.py

```diff
@@ -0,0 +1,27 @@
+#! /usr/bin/env python
+
+import rospy
+
+from scenario_test_tools.scriptable_move_base import ScriptableMoveBase
+
+from move_base_msgs.msg import MoveBaseAction, MoveBaseResult
+
+
+if __name__ == "__main__":
+    rospy.init_node('move_base_fake')
+
+    succeeded = MoveBaseResult()
+
+    move_base = ScriptableMoveBase('/move_base', MoveBaseAction, result_delay=5)
```

Comment:
I think we should have this configurable (as a parameter or arg)

Reply:
topics can be remapped, no need for a parameter